frontend-react/src/hooks/network/History/DeliveryHooks.ts

```diff
@@ -65,6 +68,11 @@ const useOrgDeliveries = (service?: string) => {
 
     const fetchResults = useCallback(
         (currentCursor: string, numResults: number) => {
+            // HACK: return empty results if requesting as an admin
+            if (activeMembership?.parsedName === Organizations.PRIMEADMINS) {
+                return Promise.resolve<RSDelivery[]>([]);
```

Comment:
This will negate the need for [lines 56-58](https://github.com/CDCgov/prime-reportstream/pull/7928/files#diff-57a7939d0bedc4fa4cbe35c6e3cc2a4a0671869ac0aad27cb2df3ef7a5d8aacbR56-R58) which adapted any Admin to the "ignore" org.

Also wondering if there is any way we can generalize this functionality for the future, since services (senders/receivers) will be driven by the organization name. Maybe once that change merges, we could have a `withServices` HOC/HOF to render a fallback UI?

Reply:
My thought here was to that they're two separate entities -- `PrimeAdmins` with no services and `ignore` as a "legitimate" Organization with services.  If we want to always make `PrimeAdmins` map to `ignore`, then shouldn't we get rid of `PrimeAdmins` entirely and just handle all admin session with `ignore`?  Otherwise, they should be treated as separate entities entirely.  I can update this in any case, but I feel like our logic around this is unintuitive right now 🤔

And yep, that's what I noted in the pull request description:

>  I think going forward, we can make a generic useOrganizationQuery hook (or whatever) that'll automatically be disabled if the user is logged in as an admin without impersonating an Organization. There are a lot of layers with our fetching that in my opinion should be untangled, but that's out of the scope of this pull request.

The changes in this pull request are intended to be minimally intrusive and temporary workarounds while the react-query migration is still underway.  I think adding a HOC at this point would be furthering the workaround logic when we could have an easier solution with a single place where we set `enabled: false` in our react-query queries if an admin isn't impersonating an Organization for Organization-specific requests.  But again, that's dependent on the migration to react-query.  What are your thoughts?

Reply:
>If we want to always make PrimeAdmins map to ignore, then shouldn't we get rid of PrimeAdmins entirely and just handle all admin session with ignore?

Oh, my comment is saying the antithesis of this: we _don't ever_ want to map one org to another, admin or otherwise, imo. :) And your change here on line 72 is going to negate any need for that conditional above. What's happening in this, I think, is:

- Sees I'm admin, "ah just change it in the outbound network call to `ignore`"
- Sees I'm admin, "ah don't send any outbound network call"

Reply:
Ah, got it -- I think we're on the same page then.  If I understand you correctly, we should really remove [this hook](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/hooks/UseAdminSafeOrganizationName.ts) and any associated/similar logic.  If so, I can update so as to _not_ map to `ignore` here and make a ticket to clean up other places we're using this logic.